internal/pkg/service/buffer/worker/service/conditions.go

```diff
@@ -113,11 +113,13 @@ func (c *checker) check(ctx context.Context) {
 		}
 
 		// Check import conditions
-		if met, reason := cdn.Evaluate(now, sliceKey.FileKey.OpenedAt(), fileStats.Total); met {
-			if err := c.swapFile(sliceKey.FileKey, reason); err != nil {
-				c.logger.Error(err)
+		if now.Sub(sliceKey.FileKey.OpenedAt()) >= c.config.MinimalImportInterval {
+			if met, reason := cdn.Evaluate(now, sliceKey.FileKey.OpenedAt(), fileStats.Total); met {
+				if err := c.swapFile(sliceKey.FileKey, reason); err != nil {
+					c.logger.Error(err)
+				}
+				continue
```

Comment:
Toto sa kontroluje uz kazdych `5s`, statistiky sa beru z `L1` cache, ktora ma par `ms` delay oproti stavu v DB (ale to nevadi).

Aby sme obmedzili interval medzi improtami, tak som pridal dalsiu podmienku.